packages/editor/src/functions/projectFunctions.tsx

```diff
@@ -38,10 +38,11 @@ import { EditorControlFunctions } from './EditorControlFunctions'
  */
 export const getProjects = async (): Promise<ProjectType[]> => {
   try {
-    const { data } = await API.instance.client.service(projectPath).find({
-      query: { allowed: true }
-    })
-    return data
+    const projects = (await API.instance.client.service(projectPath).find({
+      query: { allowed: true },
+      paginate: false
```

Comment:
Again its a client code so you cant use paginate: false here unless you implement workaround in backend.

Also aur service is always returning array.